File: MODUL 6/Bahasa Pyhton/Prak603_2410817310006_AhmadLuthfiMaulana.py
baris1, baris2 = map(int, input().split())

if baris1 != baris2:
    print("Jumlah tidak sama")
else:
    bil1 = list(map(int, input().split()))
    bil2 = list(map(int, input().split()))

    for i in range(baris1):
        print(bil1[i] * bil2[i], end=' ')

File: MODUL 6/Bahasa C/Prak603_2410817310006_AhmadLuthfiMaulana.c
#include <stdio.h>

int main() {
    int n1, n2;

    scanf("%d %d", &n1, &n2);

    if (n1 != n2) {
        printf("Jumlah tidak sama\n");
    } else {
        int bil1[n1], bil2[n2];

        for (int i = 0; i < n1; i++) {
            scanf("%d", &bil1[i]);
        }

        for (int i = 0; i < n2; i++) {
            scanf("%d", &bil2[i]);
        }

        for (int i = 0; i < n1; i++) {
            printf("%d ", bil1[i] * bil2[i]);
        }
        printf("\n");
    }

    return 0;
}